@xen-orchestra/backups/_runners/VmsXapi.js

```diff
@@ -19,6 +19,7 @@ const DEFAULT_XAPI_VM_SETTINGS = {
   concurrency: 2,
   copyRetention: 0,
   deleteFirst: false,
+  diskPerVmConcurrency: 2,
```

Comment:
Due to how we handle exports vs imports, I'm a bit worried of this change, I think we should keep it unlimited by default.